File: LeetCode/Python/Easy/867-Transpose-Matrix/solution.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        rows: int=len(matrix)

        columns: int=len(matrix[0])

        transposed: List[List[int]]=[[0 for row in range(0, rows)] for column in range(0, columns)]

        for row in range(0, rows):
            for column in range(0, columns):
                transposed[column][row]=matrix[row][column]

        return transposed

File: LeetCode/Rust/Easy/867-Transpose-Matrix/solution.rs
impl Solution
{
    pub fn transpose(matrix: Vec<Vec<i32>>) -> Vec<Vec<i32>>
    {
        let mut result: Vec<Vec<i32>>=Vec::new();

        let rows: usize=matrix.len();

        let columns: usize=matrix[0].len();

        result=vec![vec![0; rows]; columns];

        for row in 0..rows
        {
            for column in 0..columns
            {
                result[column][row]=matrix[row][column];
            }
        }

        result
    }
}